Include Entrust Root Certificate Authority

colleenbprice opened the pull request:
When I tried to develop some more with xero, it was throwing a 'certificate verify failed' error.  This commit from the waynerobinson/xeroizer repository fixes it.

danrasband commented:
@colleenbprice, all tests are passing, and this includes your payroll-us changes. There's probably a bunch of cleanup work we'll want to do before announcing this, but I think we're good to merge this into master. What do you think?

colleenbprice commented:
Sure

jstirk committed:
Include Entrust Root Certificate Authority

Thumbprint: b3 1e b1 b7 40 e3 6c 84 02 da dc 37 d4 4d f5 d4 67 49 52 f9

Fetched from
https://www.entrust.com/get-support/ssl-certificate-support/root-certificate-downloads/

danrasband commented:
👍

colleenbprice commented:
I just realized the 3.0.0 tag, which our Gemfile is pointing to, is on a different branch than master. What is our process for this?

danrasband commented:
Ideally we'll create a new tag, and point our Gemfile to that, or even more ideally, we would create our own gem on rubygems and create our own versions.
